scripts/jenkins-win64.bat

```diff
@@ -114,4 +114,9 @@ if ERRORLEVEL 1 (
 )
 echo Generating install instructions
 call %appdata%\npm\markdown-pdf install_instructions_win.md
+
+echo Generate release notes
+call %appdata%\npm\markdown-pdf RELEASE_NOTES.md
```

Comment:
@johankwarnmarksymphony Please move this to `build-win64.bat` file, we don't need `jenkins-win64` anymore. 

@mattias-symphony's WIX installer logic is part of `build-win64.bat` now.